src/docfx/lib/msgraph/MicrosoftGraphAccessor.cs

```diff
@@ -90,5 +92,21 @@ private async Task<T> SendRequest<T>(string api, Func<Task<T>> func)
                 _syncRoot.Release();
             }
         }
+
+        private static X509Certificate2 GetCertificateFromStoreByThumbprint(string certThumbprint)
+        {
+            using var store = new X509Store(StoreLocation.LocalMachine);
+            store.Open(OpenFlags.ReadOnly);
+
+            var certCollection = store.Certificates;
+            var currentCerts = certCollection.Find(X509FindType.FindByTimeValid, DateTime.Now, false);
+            var matchedCerts = currentCerts.Find(X509FindType.FindByThumbprint, certThumbprint, false);
```

Comment:
validOnly should be set to true. Please also make it explicit: `validOnly: true`. With that, we probably don't need another `Find` with `X509FindType.FindByTimeValid`